hooks/node_handlers/base_import_handler.py

```diff
@@ -301,6 +596,11 @@ def _refresh_file_path(self, node):
         if publish_data_str:
             publish_data = json.loads(self._escape_publish_data(publish_data_str))
             self._refresh_file_path_from_publish_data(node, publish_data)
+        else:
+            input_parm = node.parm(self.INPUT_PARM)
+            input_parm.lock(False)
+            input_parm.set("No publish selected")
```

Comment:
You can probably turn "No publish selected" into a public class attribute, like `NO_PUBLISH_PATH`, so it is easier to query, change, use in sub-class/hook and refactor